src/coreclr/tools/r2rtest/ProcessRunner.cs

```diff
@@ -353,6 +360,7 @@ public bool IsAvailable(ref int progressIndex, ref int failureCount)
             WriteLog(failureMessage);
 
             Console.Error.WriteLine(failureMessage + $": {processSpec}");
+            Console.Error.Write(_outputCapture.ToString());
```

Comment:
should this be writeline?

Reply:
I believe not - output capture gets filled in by lines (using the <code>AppendLine</code> method) so it always ends with a newline as an invariant (unless it's empty).